extension/chrome/elements/passphrase.ts

```diff
@@ -75,19 +75,18 @@ View.run(class PassphraseView extends View {
     $('.action_ok').click(this.setHandler(() => this.submitHandler()));
     $('#lost-pass-phrase').click(this.setHandler((el, ev) => {
       ev.preventDefault();
+      // tslint:disable-next-line:no-floating-promises
```

Comment:
While it's a minor thing here, you can fix it better by appending `.catch(Catch.reportErr)`